espanol/canudos.md

```diff
@@ -73,6 +73,8 @@ accediendo a un [estado de meditación transcendente](transcender/transcender.md
 
 [Se arrodilla y...](rogando/rogando.md)
 
+[Se ve como aparece Homero y grita](homero/homero.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.